ui/desktop/app/templates/application.hbs

```diff
@@ -155,4 +156,60 @@
       </M.Footer>
     </Hds::Modal>
   {{/if}}
-</PendingConfirmations>
\ No newline at end of file
+</PendingConfirmations>
+
+{{#if this.showSignoutModal}}
```

Comment:
I don't think you actually need two separate modals, you can just use one modal and have the handlers handle the logic for what to do when confirming or canceling and use your two variables to handle what text to display and when the modal should be open.